go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/mackerelio/mkr
 
-go 1.20
+go 1.21.0
```

Comment:
the workflow already uses Go 1.21.x https://github.com/mackerelio/mkr/blob/0fd9c02963d137ab3c98cb2dfbffb06eefa32f2e/.github/workflows/ci.yml#L31